polymath/matrix3.py

```
@@ -813,7 +824,7 @@ def mean(self, axis=None, *, recursive=True, builtins=None, dtype=None, out=None
 
         raise TypeError('Matrix3.mean() is not supported')
 
-    def __getstate__experimental(self):
+    def __getstate__experimental(self):  # pragma: no cover
```

Ruff (0.14.7)

Missing return type annotation for private function `__getstate__experimental`

(ANN202)